app/views/sessions/new.html.erb

```diff
@@ -1,7 +1,22 @@
 <h1 class="govuk-heading-xl">
   Sign in
 </h1>
-
+<p class="govuk-body">Use this product if you’re a delivery officer or caseworker at the Department for Education.</p>
+<p class="govuk-body">You can use this product to manage work that helps DfE to convert schools to academies.</p>
 <%= form_tag("/auth/azure_activedirectory_v2", method: "post") do %>
   <%= button_tag(t("sign_in.button", type: :submit), class: "govuk-button") %>
 <% end %>
+<h2 class="govuk-heading-m">Register for access</h2>
+<p class="govuk-body">If you do not have access you can <a href="https://forms.office.com/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-X7F89QcWu5CjlJXwF0TVktUMEpZSU5aTEVYVlIyRE1JSjdMREtIVDhWQiQlQCN0PWcu" class="govuk-link" rel="noreferrer noopener" target="_blank">sign up to use Complete conversions, transfers and changes (opens in new tab)</a>.</p>
+<p class="govuk-body">To register, you’ll need to provide your:</p>
+<ul class="govuk-list govuk-list--bullet">
+  <li>full name</li>
+  <li>@‌education.gov.uk email address</li>
+  <li>team information</li>
+  <li>job title</li>
```

Comment:
Does the form ask for this? Is that how we work out the if they are a 'lead'?

Reply:
Yeah, it asks your name, email, which team you're in and what your job title is (or grade where folks just get called the G6 for a particular region for example). Once we know that we can work out if they're a delivery officer, caseworker or RCS team lead/somebody who needs to assign work.